lib/utils.ts

```diff
@@ -43,7 +44,30 @@ export const configPath = configPathExpected || configPathRoot
 /**
  * configuration
  */
-export const config: Config = { ...defaultConfig, ...getUserConfig() }
+export const config: Config = getUserConfig()
+
+/**
+ * Gets a configuration value or prompt for runing `init` command again
+ */
+export function getConfigOrPrompt<Key extends keyof Config>(key: Key): Config[Key] {
+  const config = getUserConfig({ withDefault: false })
+
+  if (config[key]) return config[key]
+
+  const defaultValue = defaultConfig[key]
+
+  console.log(
+    [
+      `${chalk.cyan.bold(key)} is not set in ${chalk.yellow(configFileName)}`,
+      `By default it is set to ${chalk.cyan(defaultValue)}`,
+      `You can change it in ${chalk.yellow(configFileName)} or run ${chalk.cyan(
+        'linguify init'
+      )} again to configure missing config`
+    ].join('\n')
+  )
+
+  return defaultValue
+}
```

Comment:
This utility to keep the dev up-to-date with the config updates when releasing new configurations like this.

Reply:
This utility can replace `linguifyValidation()`

Reply:
the validations in `linguifyValidation()` is not only for checking if it is set or not, so it is still needed